File: kotlin-ide/j2k/old/testData/fileOrElement/constructors/fieldsInitializedFromParamsAnnotations.java
import java.lang.SuppressWarnings;

class C {
    @Deprecated private final int p1;
    private final int myP2;
    @SuppressWarnings("x") public int p3;

    public C(int p1, @Deprecated int p2, @Deprecated int p3) {
        this.p1 = p1;
        myP2 = p2;
        this.p3 = p3;
    }
}

File: nj2k/testData/newJ2k/constructors/fieldsInitializedFromParamsAnnotations.kt
// ERROR: This annotation is not applicable to target 'value parameter' and use site target '@field'
// ERROR: This annotation is not applicable to target 'value parameter' and use site target '@param'
// ERROR: This annotation is not applicable to target 'value parameter' and use site target '@param'
internal class C(@field:Deprecated("") private val p1: Int, @param:Deprecated("") private val myP2: Int, @param:Deprecated("") var p3: Int) 